ee-feature-pack/galleon-common/src/main/resources/layers/standalone/web-console/layer-spec.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="web-console">
+    <dependencies>
+        <layer name="secure-management"/>
```

Comment:
Replace this by `management` once https://github.com/wildfly/wildfly-core/pull/4296 is merged

Reply:
Actually this can be done once https://github.com/wildfly/wildfly-core/pull/4295 is merged and available, and it is.